tedana/selection/selection_utils.py

```diff
@@ -160,9 +160,9 @@ def change_comptable_classifications(
         dont_warn_reclassify=dont_warn_reclassify,
     )
 
-    selector.component_status_table[f"Node {selector.current_node_idx}"] = (
-        selector.component_table["classification"]
-    )
+    selector.component_status_table[
+        f"Node {selector.current_node_idx}"
+    ] = selector.component_table["classification"]
```

Comment:
These stylistic changes may come from an old version of black, based on the style error in https://github.com/ME-ICA/tedana/pull/1031.

Reply:
This PR has an older version in `.pre-commit-config.yaml` that was causing this problem.